Week01/Problem01/klai_01.py

```diff
@@ -0,0 +1,2 @@
+l = [x for x in range(1,1000) if x%3 == 0 or x%5 == 0]
+print sum(l)
```

Comment:
Careful, here.  You're using Python 2, so it's fine, but this **will** break for anyone using Python 3+.  The print function has been back-ported to Python 2, however, so you can legally write
```
print(sum(l))
```
without your interpreter complaining.  Just something to keep in mind...